### DISCUSSION TITLE
Asus VE276Q: volume doesn't work

### DISCUSSION THREAD

**Discussion by BoD** [MonitorControl/MonitorControl#1152] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume is not working on my VE276Q.

I've tried `m1ddc`, and it's also not working. It's returning strange values though.

```
$ ./m1ddc display 1 get volume
110
$ ./m1ddc display 1 max volume
-128
```

Luminance is also returning these values but it is working (with m1ddc and MonitorControl).

### Expected behavior

Volume should work (if possible!) :)

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5 Monterey
- Mac model: MacBook Pro (16-inch, 2021) Apple M1 Max
- MonitorControl version: 4.1.0
- Monitor(s): Asus VE276Q
- Monitor Cable(s)/Connection(s): USB-C to HDMI
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Some displays simply do not support volume control over DDC. The DDC communication is working so the problem is most probably the display firmware. :( I am afraid not much can be done about this.

**Comment by BoD** [comment]
Yeah I guessed it was probably that (plus this model is fairly old too). Thanks for taking the time to look at it! 🙏
